src/runtime/components/forms/Toggle.vue

```diff
@@ -51,7 +53,15 @@ export default defineComponent({
       default: null
     },
     modelValue: {
-      type: Boolean,
+      type: AnyType,
+      default: false
+    },
+    selectedValue: {
```

Comment:
Actually, it would be better to call it `onValue` and `offValue` like for the icons I guess.

Reply:
Yes, that's correct. A toggle component is commonly used to switch something on or off, such as a setting or feature within a user interface.

Reply:
> Actually, it would be better to call it `onValue` and `offValue` like for the icons I guess.

I replaced the prop names 'onValue' and 'offValue' with 'selectedValue' and 'unselectedValue,' respectively. The initial change was made to enhance the clarity and consistency of the component's props. However, I realized that using props that start with 'on' causes Vue to automatically detect an emit, which is a problem for documentation purposes. Therefore, I decided to revert the change and use the original prop names to avoid any conflicts and ensure proper functionality.